documents/src/pages/elements/select.md

```diff
@@ -15,49 +15,35 @@ language_tabs: [javascript, typescript]
 ```css
 section {
   height: 250px;
-  padding: 0 3px;
 }
 ef-select {
-  margin-right: 5px;
+  width: 200px;
+  margin-left: 4px;
 }
 ```
 ```html
 <section>
-  <ef-select placeholder="Pick item..." opened>
-    <ef-item type="header">Drinks</ef-item>
-    <ef-item value="1">Cola</ef-item>
-    <ef-item value="2" disabled>Apple Juice</ef-item>
-    <ef-item value="3">Iced Tea</ef-item>
-    <ef-item type="header">Ice Cream</ef-item>
-    <ef-item value="4">Vanilla</ef-item>
-    <ef-item value="5">Chocolate</ef-item>
-    <ef-item value="6">Honey &amp; Walnut</ef-item>
-    <ef-item value="7">Raspberry</ef-item>
-  </ef-select>
-  <ef-select placeholder="Disabled..." disabled></ef-select>
-  <ef-select placeholder="Default selected...">
-    <ef-item type="header">Drinks</ef-item>
-    <ef-item value="1">Cola</ef-item>
-    <ef-item value="2" disabled>Apple Juice</ef-item>
-    <ef-item selected value="3">Iced Tea</ef-item>
-    <ef-item type="header">Ice Cream</ef-item>
-    <ef-item value="4">Vanilla</ef-item>
-    <ef-item value="5">Chocolate</ef-item>
-    <ef-item value="6">Honey &amp; Walnut</ef-item>
-    <ef-item value="7">Raspberry</ef-item>
-  </ef-select>
-  <ef-select disabled>
-    <ef-item selected>Lemonade</ef-item>
+  <label for="desserts">Desserts </label>
+  <ef-select id="desserts" placeholder="Pick your preferred desserts" opened>
```

Comment:
The placeholder is always hidden because of the selected option. I think it should be removed.

Reply:
Updated.